.github/workflows/codeql-analysis.yml

```diff
@@ -1,14 +1,3 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
 name: "CodeQL"
 
 on:
```

Comment:
I would include enable manual start - for convenience if anybody wants to invoke it manually

```
  workflow_dispatch:
    branches:
      - 'develop'
```